Makefile

```diff
@@ -382,7 +384,7 @@ endif
 WARNINGS = all error strict-prototypes missing-prototypes nested-externs \
 	missing-declarations undef pointer-arith no-nonnull declaration-after-statement
 
-CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding \
+CFLAGS += --std=c99 -nostdlib -nostdinc -ffreestanding -fms-extensions \
```

Comment:
I was build with my newer gcc, but toolchain crashed (CONFIG_PRINTING relative I think).then I test my old gcc and got this issue,so I fixed.

if you don't like, just don't merge.

Thanks